Sources/CorePayments/Networking/AnalyticsService.swift

```diff
@@ -33,24 +33,25 @@ public struct AnalyticsService {
     ///
     /// Sends analytics event to https://api.paypal.com/v1/tracking/events/ via a background task.
     /// - Parameter name: Event name string used to identify this unique event in FPTI.
-    public func sendEvent(_ name: String) {
+    public func sendEvent(_ name: String, correlationID: String? = nil) {
         Task(priority: .background) {
-            await performEventRequest(name)
+            await performEventRequest(name, correlationID: correlationID)
         }
     }
     
     // MARK: - Internal Methods
     
     /// Exposed to be able to execute this function synchronously in unit tests
-    func performEventRequest(_ name: String) async {
+    func performEventRequest(_ name: String, correlationID: String? = nil) async {
```

Comment:
Nit - update docstring for this new param

Reply:
Good call - done: [PR feedback - add docstrings for correlationID](https://github.com/paypal/paypal-ios/pull/208/commits/bcc835e12572d05e9987f4386d64ab229c1f64a8)